UI/window-basic-main.cpp

```diff
@@ -2670,6 +2670,20 @@ void OBSBasic::CreateHotkeys()
 	LoadHotkeyPair(replayBufHotkeys, "OBSBasic.StartReplayBuffer",
 		       "OBSBasic.StopReplayBuffer");
 
+	auto replayBufferCallback = [](void *data, obs_hotkey_id,
+				       obs_hotkey_t *, bool pressed) {
+		OBSBasic &basic = *static_cast<OBSBasic *>(data);
```

Comment:
Personally not a fan of this syntax. Not sure what others say, I prefer just using pointers directly (which is how it’s done in most other places).